pom.xml

```diff
@@ -64,12 +64,12 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
-        <scala.version>2.11.12</scala.version>
-        <scala.version.short>2.11</scala.version.short>
-        <akka.version>2.4.20</akka.version>
-        <akka.http.version>10.0.11</akka.http.version>
+        <scala.version>2.13.1</scala.version>
+        <scala.version.short>2.13</scala.version.short>
+        <akka.version>2.6.4</akka.version>
+        <akka.http.version>10.1.11</akka.http.version>
         <sttp.version>1.7.2</sttp.version>
-        <bitcoinlib.version>0.17</bitcoinlib.version>
+        <bitcoinlib.version>0.18-SNAPSHOT</bitcoinlib.version>
```

Comment:
Placeholder comment so that we don't forget to update that to a release version of bitcoinlib

Reply:
Agreed. @sstone 🙏

Reply:
Yes I'll add a comment when it's available on maven central

Reply:
bitcoin-lib_2.13 0.18 is now available on maven central